packages/api/lib/testUtils.js

```diff
@@ -510,14 +510,15 @@ const createRuleTestRecords = async (context, ruleParams) => {
   } = context;
   const originalRule = fakeRuleFactoryV2(ruleParams);
 
-  const ruleWithTrigger = await ruleModel.createRuleTrigger(originalRule);
-  const insertPgRecord = await translateApiRuleToPostgresRule(ruleWithTrigger, testKnex);
-  const originalDynamoRule = await ruleModel.create(ruleWithTrigger);
+  const dynamoRuleWithTrigger = await ruleModel.createRuleTrigger(originalRule);
+  const originalDynamoRule = await ruleModel.create(dynamoRuleWithTrigger);
+  const insertPgRecord = await translateApiRuleToPostgresRule(originalDynamoRule, testKnex);
+
   const [ruleCumulusId] = await rulePgModel.create(testKnex, insertPgRecord);
   const originalPgRecord = await rulePgModel.get(
     testKnex, { cumulus_id: ruleCumulusId }
   );
-  await indexRule(esClient, ruleWithTrigger, process.env.ES_INDEX);
+  await indexRule(esClient, originalDynamoRule, process.env.ES_INDEX);
```

Comment:
Interesting, and nice catch/protect against future badness.       To confirm, the reason this didn't blow up the rule units is that in most cases we're not relying on the trigger Arn for the OG record,
(e.g. cases where it isn't supposed to write, or it's supposed to be updated anyway), etc?

Reply:
I think the reason it didn't matter is there is little if any difference between `dynamoRuleWithTrigger` and `originalDynamoRule`, so using either as the basis for indexing to ES is unlikely to produce any issues. But it seemed perhaps safer to index exactly what got created in ES in case somehow the logic changes.

It could be argued we should go the other way though, since P2 and thus PG is now the basis of truth:

- generating a PG record fixture
- create PG record
- translate PG record -> API
- save API record to dynamo
- save created dynamo record to ES

Reply:
Scratch that. That'll have to wait for phase 3 because `createRuleTrigger` currently accepts a dynamo rule so it's a pain to make that change without translating from PG to API and back again